openhands/server/listen.py

```diff
@@ -5,28 +5,10 @@
 import time
```

Comment:
Ruff formatting is failing on `listen.py`. We should run ruff to fix the formatting.